auth/src/main/java/io/grpc/auth/ClientAuthInterceptor.java

```diff
@@ -64,12 +64,11 @@
 
   private Metadata cached;
   private Map<String, List<String>> lastMetadata;
-  // TODO(louiscryan): refresh token asynchronously with this executor.
-  private Executor executor;
 
-  public ClientAuthInterceptor(Credentials credentials, Executor executor) {
+  public ClientAuthInterceptor(
+      Credentials credentials, @SuppressWarnings("unused") Executor executor) {
     this.credentials = Preconditions.checkNotNull(credentials);
-    this.executor = Preconditions.checkNotNull(executor);
+    // TODO(louiscryan): refresh token asynchronously with this executor.
```

Comment:
When we fix the TODO, wouldn't we go back to saving the executor? I don't see it's necessary for now to make this change.